grizzly/users/meta/request_logger.py

```diff
@@ -12,12 +12,13 @@
 from jinja2 import Template
 
 from ...task import RequestTask
-from ...transformer import JsonBytesEncoder
 from ...types import HandlerContextType, RequestDirection
 from ...utils import merge_dicts
 from .response_event import ResponseEvent
 from .context_variables import ContextVariables
 
+from grizzly_extras.transformer import JsonBytesEncoder
```

Comment:
There is a identical `JsonBytesEncoder` in `grizzly_extras.async_message`, since we're moving `.transformer` from `grizzly.` it makes sense to remove that one, and use the one in `grizzly_extras.transformer` instead (implicit change by this PR).